cmd/cluster/core/create.go

```diff
@@ -241,6 +243,14 @@ func createCommonFixture(ctx context.Context, opts *CreateOptions) (*apifixtures
 		}
 	}
 
+	if opts.DefaultDual {
+		if len(opts.AgentPlatform.APIServerAddress) <= 0 || len(opts.AgentPlatform.AgentNamespace) <= 0 {
+			return nil, fmt.Errorf("--default-dual is only supported on Agent platform")
```

Comment:
If this is only supported on the Agent platform, should the flag just be moved to the Agent create.go rather than the general cluster create?

Reply:
I need the service-cidr and cluster-cidr to make them MutuallyExclusive with the default-dual flag and this is only in the general. But even If I move it to the agent.go/create it can be used with other providers also (I've tested that already, this is because how we are building the CLI)